notification-center/introduction.mdx

```diff
@@ -6,37 +6,31 @@ icon: "house"
 
 The notification center is a component, also sometimes referred to as a widget that furnishes your web application with a beautiful and smooth in-app notification experience for users.
 
-In-app notifications are different from push notifications in the sense that in-app notifications are consumed by the user in your app itself, whether push notifications are shown on the device level.
+In-app notifications are different from push notifications in the sense that in-app notifications are consumed by the user in your app itself, whether push notifications are shown on the device level. Read more about push channel and supported providers [here](/channels-and-providers/push/introduction)
```

Comment:
Please let's avoid the use of `here` for links. We are trying to eradicate all `here` from the documentation for better accessibility to screen readers and different type of audiences.

So in this case, it can be [Read more about the push channel and supported providers](/channels-and-providers/push/introduction)